packages/node_modules/nav-frontend-lenkepanel/src/Lenkepanel-base.tsx

```diff
@@ -1,15 +1,22 @@
-import PT from 'prop-types';
-import React, { Component } from 'react';
-import classnames from 'classnames';
-import 'nav-frontend-lenkepanel-style'; // eslint-disable-line import/extensions
-
-const cls = (className) => classnames('lenkepanel', className);
-/**
- * Lenkepanel
- */
-class LenkepanelBase extends Component {
+import * as PT from 'prop-types';
+import * as React from 'react';
+import * as classNames from 'classnames';
+import 'nav-frontend-lenkepanel-style';
+
+const cls = (className) => classNames('lenkepanel', className);
+
+export interface LenkepanelBaseProps {
+    className?: string;
+    href: string;
+    children: React.ReactNode | React.ReactNode[];
+    linkCreator?: Function;
```

Comment:
ikke ønskelig å bruke `Function`. Fint å bruke noe mer spesifikt som `(arg1: string) => void`.

Note to self: dra inn tslint og lag en regel om dette